crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -930,6 +952,10 @@ impl AuthorityPerEpochStore {
             &self.tables.pending_certificates,
             [(*certificate.digest(), certificate.clone().serializable())],
         )?;
+        let batch = batch.insert_batch(
```

Comment:
This may no longer be idempotent, in that when processing the same cert (but with slightly different user sig) would end up with different result.
Could you 1) explain why this is guaranteed to be done once per tx 2) have a debug assert to make sure we never double insert?

Reply:
Added assert and comment.
However, there is nothing special about this particular table, comparing to other tables in the handling consensus certificate flow - all writes here are done in a single batch, with last write putting an entry into table indicating that certificate for tx digest was processed